spacepy/poppy.py

```diff
@@ -420,7 +396,7 @@ def plot(self, figsize=None, dpi=80, asympt=True, show=True, norm=True,
                 plt.ylabel(
                     'Normalized Association Number n(u, h={0}) / n({1}, h={0})'.format(
                     self.winhalf,
-                    r'$\mathrm{u\rightarrow\infty}$'
+                    r'$\\mathrm{u\rightarrow\infty}$'
```

Comment:
This is a raw string so the backslash doesn't need to be escaped.

Reply:
resolved